helpers/parser.py

```diff
@@ -3,9 +3,54 @@
                              QPushButton, QVBoxLayout, QWidget)
 
 from helpers import config
+from datetime import datetime
 
 
-class ParserWindow(QFrame):
+class Parser:
```

Comment:
I'm on the fence about this, but it's growing on me. Maybe there would be other things that don't need a window? IDK, still thinking

Reply:
It does solve the problem neatly for any non-window parser, now or in the future.